database/migrations/2019_11_07_230457_add_hr_advisor_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddHrAdvisorTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('hr_advisors', function (Blueprint $table) {
+            $table->increments('id');
+            $table->integer('user_id')->unsigned()->unique()->index();
+            $table->integer('department_id')->unsigned()->nullable()->index();
```

Comment:
We can add a foreign key constraint here, to ensure only valid department ids can be entered.